### PR TITLE
Enable serial testing

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tests/synchronizer.toit [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  client-socket-latch_/monitor.Latch? := monitor.Latch
+  task_/Task? := null
+
+  constructor:
+    network_ = net.open
+    server-socket_ = network_.tcp-listen 0
+    task_ = task::
+      client-socket_ = server-socket_.accept
+      client-socket-latch_.set client-socket_
+      task_ = null
```
Comment (kasperl) [inline tests/synchronizer.toit:26]:
```suggestion
      try:
        client-socket_ = server-socket_.accept
        client-socket-latch_.set client-socket_
      finally:
        task_ = null
```

File: tests/synchronizer.toit [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  close:
+    if task_:
+      task_.cancel
+      task_ = null
```
Comment (kasperl) [inline tests/synchronizer.toit:31]: I'd drop this.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/synchronizer.toit (hunk 1 above)
Comment (floitsch) [reply tests/synchronizer.toit]: done.

File: tests/synchronizer.toit (hunk 2 above)
Comment (floitsch) [reply tests/synchronizer.toit]: done.

**Pull request merged**